packages/components/src/components/SolutionsMenu.styles.tsx

```diff
@@ -0,0 +1,56 @@
+import tw, { css, styled } from 'twin.macro';
+
+export const Category = styled.div(() => [
+  tw`mb-6 last:mb-0`,
+  css`
+    h3 {
+      ${tw`w-full mt-1 mb-5 font-normal text-base text-gray-600 dark:text-gray-400 text-black`}
+    }
+  `,
+]);
+
+export const List = styled.div(() => [tw`flex flex-wrap`]);
+
+export const Item = styled.a(() => [
+  tw`box-border flex items-center w-full my-1 md:(w-1/2 m-0) py-2 px-2 rounded-lg outline-none no-underline!`,
+  tw`dark:hocus:bg-gray-700 hocus:bg-gray-100`,
+  css`
+    &:hover {
+      h4 {
+        ${tw`text-black dark:text-white`}
+      }
+    }
+    span {
+      ${tw`flex flex-col justify-center`}
+      h4 {
+        ${tw`m-0`}
+      }
+
+      h4 {
+        ${tw`font-medium text-xs dark:text-gray-500 text-gray-400`}
+      }
+    }
+  `,
+]);
+
+export const Callout = styled.div(() => [
+  tw`mt-10 p-2`,
+  css`
+    display: flex;
+    justify-content: end;
```

Comment:
```suggestion
  tw`mt-10 p-2 flex justify-end`,
  css`
```
also `justify-content: end` don't exist
<img width="536" alt="image" src="https://user-images.githubusercontent.com/7361780/165731275-096d5723-db72-4690-9c51-ac20f400a653.png">